sim/druid/shred.go

```diff
@@ -7,19 +7,29 @@ import (
 	"github.com/wowsims/sod/sim/core/proto"
 )
 
+var sodShredEffect1Mult = 0.75 // Multiply flat dmg by this
```

Comment:
Looks like https://www.wowhead.com/classic/spell=436895/s03-tuning-and-overrides-passive-druid explicitly lowers the base damage (-25%) via "Apply Aura: Modifies Effect #1's Value", and this applies after the "regular" changes.

Reply:
Yeah, I wanted to recreate what the game does, hence the explicit 0.75 multiplier for effect1. But yes, naming and placement of the vars isn't optimal, made that in a hurry :D

Order for percentage "all effects" and "effect #" mods on the server side is unknown, but because it's both multiplicative it doesn't matter, unlike for the additive one for effect 2.